packages/web-components/fast-element/src/templating/binding.ts

```diff
@@ -33,7 +33,7 @@ export interface BindingMode {
     event: BindingType;
 }
 
-export interface BindingConfig<T = any> {
+export interface BindingConfig {
```

Comment:
I think we need this generic for inferred types in actual usage scenarios. Possible to add it back?

Reply:
I've got a warning/error that it was unused, I can add it back and change the eslint config instead.

Reply:
Added it as a one line ignore.